hack/update-k8s-dep.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
Rename this to update-k8s-dep*s*.sh? Feel free to ignore if this is coming from other repos.

Reply:
Yeah I have it in other repo - I could it in a separate PR